### ISSUE TITLE
Show person and place creation forms in the Reports tab

### ISSUE THREAD

**Issue by sglangevin** [medic/cht-core#3415]
Now that we have person creation via SMS, that form shows up in the Reports tab. If a person or place is created with an xform, however, there is no easy way to see which people or places were created recently. Display all person and place creation forms in the Reports tab so that a CHW or Nurse can easily review people they have recently registered and places they have recently created.

**Comment by rmhowe** [comment]
@sglangevin is this still relevant? Maybe for 3.1?

**Comment by sglangevin** [comment]
Yes, still relevant. I'd put it as a nice to have for 3.1, we can see if there is time for it there.
